third_party/terraform/tests/resource_logging_billing_account_sink_test.go

```diff
@@ -221,3 +253,39 @@ resource "google_storage_bucket" "log-bucket" {
 	name     = "%s"
 }`, name, billingAccount, getTestProjectFromEnv(), bucketName)
 }
+
+func testAccLoggingBillingAccountSink_bigquery_before(sinkName, bqDatasetID, billingAccount string) string {
+	return fmt.Sprintf(`
+resource "google_logging_billing_account_sink" "bigquery" {
+  name             = "%s"
+  billing_account  = "%s"
+  destination      = "bigquery.googleapis.com/projects/%s/datasets/${google_bigquery_dataset.logging_sink.dataset_id}"
+  filter           = "logName=\"projects/%s/logs/compute.googleapis.com%%2Factivity_log\" AND severity>=ERROR"
+  include_children = true
```

Comment:
```suggestion
```
Billing account sink doesn't have this field https://www.terraform.io/docs/providers/google/r/logging_billing_account_sink.html